docs/update-help-messages.sh

```diff
@@ -18,17 +19,26 @@ function print_help_to_file() {
     local cmd
     cmd=$(echo "$*" | sed -E "s/^ +//")
 
-    # Replace spaces in the file name with dashes We print the output of
-    # `pgcopydb --help` to `pgcopydb.rst`
-    # `pgcopydb compare --help` to `compare.rst`
-    # `pgcopydb compare data --help` to `compare-data.rst`
+    # Replace spaces in the command name with dashes to generate the file name.
+    #
+    # We print the output of
+    #   `pgcopydb --help` to `pgcopydb.rst`
+    #   `pgcopydb compare --help` to `compare.rst`
+    #   `pgcopydb compare data --help` to `compare-data.rst` etc.
+    #
+    #  One exception is the `pgcopydb help` command. We print the output of
+    #  `pgcopydb help` to `help.rst` instead of supplying a `--help` argument to
+    #  the command.
     local file_path
     local help_cmd
     if [ "${cmd}" = "" ]; then
-        help_cmd="pgcopydb --help 2>&1"
+        help_cmd="$PGCOPYDB --help 2>&1"
```

Comment:
Please use `${PGCOPYDB}` notation with the `{}` as everywhere else in our shell scripts.

Reply:
Also please install a default value in the case that the environment variable has not been set.